### PR TITLE
Reduced memory leaks from loops in CreateAlphaByteString

### PR DESCRIPTION
**Pull request by marcusnaslund**
@sebastianbaginski - peer review?

### CODE REVIEW

File: source/draw/CreateAlphaByteString.ooc [hunk 1]
```diff
@@ -32,9 +32,12 @@ CreateAlphaByteString: class {
 			ipbuffer pointer[i] = buf pointer[(i * 4) + 3]
 		ip = ipbuffer pointer as Int*
 		if (ipbuffer size > 3)
-			imageArray += ip[0] toString()
-		for (i in 1 .. ipbuffer size / 4)
-			imageArray += "," + (i % 32 == 0 ? "\n" : " ") + ip[i] toString()
+			imageArray = imageArray >> ip[0] toString()
+		for (i in 1 .. ipbuffer size / 4) {
+			intString := ip[i] toString()
+			imageArray = imageArray >> "," >> (i % 32 == 0 ? "\n" : " ") >> intString
```
Comment (sebastianbaginski) [inline source/draw/CreateAlphaByteString.ooc]: Does this part `imageArray >> "," >> (i % 32 == 0 ? "\n" : " ")` looks OK with valgrind ? I never know when a string like that is a static literal and I try to free it.

**Comment by sebastianbaginski** [comment]
Looks good except the one part I'm not sure of.

File: source/draw/CreateAlphaByteString.ooc (hunk 1 above)
Comment (marcusnaslund) [inline source/draw/CreateAlphaByteString.ooc]: > I never know when a string like that is a static literal

You and me both. Thanks for the comment, I will check it.

Comment (marcusnaslund) [inline source/draw/CreateAlphaByteString.ooc]: No invalid read/writes.

**Pull request merged**